homeassistant/components/smarty/switch.py

```diff
@@ -0,0 +1,93 @@
+"""Support for Salda Smarty XP/XV Ventilation Unit Sensors."""
```

Comment:
Stale docstring

Reply:
I suppose it's better to replace it with """Platform to control a Salda Smarty XP/XV ventilation unit."""
Do you agree?

Reply:
Replaced